app/assets/javascripts/no_sunet.js

```diff
@@ -0,0 +1,43 @@
+$(document).on('click', '#no-sunet-button', function(){
+  setTimeout(function(){
+      $('#request_user_attributes_library_id').focus();
+  }, 1);
+
+  $('form').on('keypress', function(e) {
+    if(e.which == 13) {
+      $('.form-group input').checkForm(e);
+    }
+  });
+
+  $('.form-control').on('mouseenter mouseleave focus blur',function(e){
+    $('.form-group input').checkForm(e);
+  });
+
+  $('#no-sunet-submit').click(function(e) {
```

Comment:
This and the `$('form').on('keypress')` are kinda strange to me here.  Is this simply to capture the form submission (which is better handled with `$('form').on('submit')`)?

(btw, the `$('form')` selector will grab the first form in the page, if there is more than one form this may cause an issue)